src/System.ServiceModel.Http/tests/ServiceModel/BasicHttpBindingTest.cs

```diff
@@ -169,6 +169,7 @@ public static void Name_Property_Sets(string value)
     [Theory]
     [InlineData(null)]
     [InlineData("")]
+    [ActiveIssue(1449)]
```

Comment:
Can we do something like 

```
      [Theory]
      [InlineData("")]
      [InlineData(null)]
#if FEATURE_NETNATIVE
      [ActiveIssue(1449)]
#endif
```

or conversely 

```
      [Theory]
      [InlineData("")]
#if !FEATURE_NETNATIVE
      [InlineData(null)]
#endif
```

So that we don't end up indiscriminately skipping all our Theory tests will null in UWP?